frontend/src/scenes/events/PropertiesVolumeTable.tsx

```diff
@@ -1,28 +1,31 @@
 import React from 'react'
 import { useValues } from 'kea'
-import { VolumeTable, UsageDisabledWarning, EventOrPropType } from './EventsVolumeTable'
-import { Alert } from 'antd'
+import { VolumeTable, UsageDisabledWarning } from './EventsVolumeTable'
+import { Alert, Skeleton } from 'antd'
 import { preflightLogic } from 'scenes/PreflightCheck/logic'
-import { teamLogic } from 'scenes/teamLogic'
+import { propertyDefinitionsLogic } from './propertyDefinitionsLogic'
 
 export function PropertiesVolumeTable(): JSX.Element | null {
-    const { currentTeam } = useValues(teamLogic)
     const { preflight } = useValues(preflightLogic)
-    return currentTeam?.event_properties_with_usage ? (
+    const { propertyDefinitions, loaded } = useValues(propertyDefinitionsLogic)
+
+    return loaded ? (
         <>
             {preflight && !preflight?.is_event_property_usage_enabled ? (
                 <UsageDisabledWarning tab="Properties Stats" />
             ) : (
-                currentTeam?.event_properties_with_usage[0]?.volume === null && (
+                propertyDefinitions[0].volume_30_day === null && (
```

Comment:
Hmm... I do remember adding a change to the backend PR that sets all of these to `0` (not `null`). Maybe worth reverting that...

Reply:
I added the `null`s back